gx_suite_definitions/gene_info.ipynb

```diff
@@ -0,0 +1,433 @@
+    "validator.expect_column_values_to_not_be_null(\"alias\")\n",
+    "validator.expect_column_values_to_have_list_members_of_type(column=\"alias\", member_type=\"str\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# is_igap\n",
+    "validator.expect_column_values_to_be_of_type(\"is_igap\", \"bool\")\n",
+    "validator.expect_column_values_to_not_be_null(\"is_igap\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# is_eqtl\n",
+    "validator.expect_column_values_to_be_of_type(\"is_eqtl\", \"bool\")\n",
+    "validator.expect_column_values_to_not_be_null(\"is_eqtl\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# is_any_rna_changed_in_ad_brain\n",
+    "validator.expect_column_values_to_be_of_type(\"is_any_rna_changed_in_ad_brain\", \"bool\")\n",
+    "validator.expect_column_values_to_not_be_null(\"is_any_rna_changed_in_ad_brain\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# rna_brain_change_studied\n",
+    "validator.expect_column_values_to_be_of_type(\"rna_brain_change_studied\", \"bool\")\n",
+    "validator.expect_column_values_to_not_be_null(\"rna_brain_change_studied\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# is_any_protein_changed_in_ad_brain\n",
+    "validator.expect_column_values_to_be_of_type(\"is_any_protein_changed_in_ad_brain\", \"bool\")\n",
+    "validator.expect_column_values_to_not_be_null(\"is_any_protein_changed_in_ad_brain\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# protein_brain_change_studied\n",
+    "validator.expect_column_values_to_be_of_type(\"protein_brain_change_studied\", \"bool\")\n",
+    "validator.expect_column_values_to_not_be_null(\"protein_brain_change_studied\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# target_nominations\n",
+    "validator.expect_column_values_to_be_of_type(\"target_nominations\", \"str\")\n",
+    "with open(\"../src/agoradatatools/great_expectations/gx/json_schemas/gene_info/target_nominations.json\", \"r\") as file:\n",
+    "    target_nominations_schema = json.load(file)\n",
+    "validator.expect_column_values_to_match_json_schema(\"target_nominations\", json_schema=target_nominations_schema, mostly=0.98)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# median_expression\n",
+    "validator.expect_column_values_to_be_of_type(\"median_expression\", \"str\")\n",
```

Comment:
Question: the median expression field is actually a `dict` I think, why are we expecting its values to be strings? Is it converted to a JSON string or something?

Reply:
That's exactly right. It is converted to a JSON parseable string so that we can use the JSON schema expectation